consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/SubmitQueryConsumerTest.java

```diff
@@ -275,26 +307,26 @@ private ISubmitQueryRequest createSubmitQueryRequest() {
         serviceRoutings.add(serviceRouting);
         targetApp.setServiceRoutings(serviceRoutings);
 
-        final Set<IBulkCustomerApplication> bulkCustomerApplications = new HashSet<IBulkCustomerApplication>();
+        final Set<IBulkCustomerApplication> bulkCustomerApplications = new HashSet<>();
         final BulkCustomerApplication bulkCustomerApplication = new BulkCustomerApplication();
         bulkCustomerApplication.setBulkCustomer(bulkCustomer);
         bulkCustomerApplication.setTargetApplication(targetApp);
         bulkCustomerApplications.add(bulkCustomerApplication);
         bulkCustomer.setBulkCustomerApplications(bulkCustomerApplications);
 
-        final ISubmitQueryRequest submitQueryRequest = new SubmitQueryRequest();
-        submitQueryRequest.setBulkCustomer(bulkCustomer);
-        submitQueryRequest.setTargetApplication(targetApp);
-        submitQueryRequest.setResultCount(1);
-        return submitQueryRequest;
+        final ISubmitQueryRequest ssubmitQueryRequest = new SubmitQueryRequest();
+        ssubmitQueryRequest.setBulkCustomer(bulkCustomer);
+        ssubmitQueryRequest.setTargetApplication(targetApp);
+        ssubmitQueryRequest.setResultCount(1);
+        return ssubmitQueryRequest;
 
     }
 
     /**
      * Need to extend the consumer class under test for overriding base class methods
      * of the getClient as it is abstract method.
      */
-    private class SubQueryConsumer extends SubmitQueryConsumer {
+    protected class SubQueryConsumer extends SubmitQueryConsumer {

```

Comment:
Again, wonder if this nested class is needed now that the mocking framework has been changed?

Reply:
Had a good play around with that today. Could see no alternative.